doc/design.md

```diff
@@ -152,17 +151,15 @@ Creates a user
 
 - format: `JSON`
 - fields:
-    * username
-	* encrypted password
-	* email address
+	* email
+    * encrypted password
```

Comment:
I don't mean to suggest that this is wrong, honestly just wondering: is there a reason why the indentation is different here for email and encrypted password?

Reply:
Do you mean before this change? I wonder if spaces and tabs are getting mixed up.

Reply:
It was a tabs and spaces thing. I just fixed those